synapse/types.py

```diff
@@ -743,6 +744,26 @@ class ReadReceipt:
     data: JsonDict
 
 
+@attr.s(slots=True, frozen=True, auto_attribs=True)
+class DeviceLists:
```

Comment:
Ah, well the `Lists` in `DeviceLists` is referring to a user's device list, rather than the lists in the Python object. Though I see how that could be confusing?

`DeviceListUpdates` sounds the best to me? Paired with an actual docstring? An instance is an object containing two sets of user IDs which indicate changes to users' device lists.